main.rb

```diff
@@ -33,12 +34,12 @@ def runCommand(command)
 	puts "Gemfile exists in working directory."
 	if OS.mac?
 		if `which rbenv`.empty?
-			runCommand("sudo gem install bundler")
+			runCommand("sudo gem install bundler #{bundler_version}")
```

Comment:
@boztopuz All our runner environments have `rbenv` installed. So, practically, this branch in the **if** is unreachable. :bulb: 

When we remove it, `OS.mac?` check also becomes obsolete since the **if** and **else** branches have same content.

Finally we can have one line gem install here. I will refactor it. :raising_hand_man: